frontend/src/components/auth/Signup.tsx

```diff
@@ -21,7 +21,7 @@ const Signup = (): React.ReactElement => {
   const role = params.get("role");
 
   if (role !== "facilitator") {
-    return <Redirect to={WELCOME_PAGE} />;
+    return <Redirect to={HOME_PAGE} />;
```

Comment:
hmm so this is the case where someone does like `/signup?role=blah` - I think we do want to redirect to `WELCOME_PAGE`